docs/api-reference/carto/fetch-map.md

```diff
@@ -41,30 +41,67 @@ fetchMap({cartoMapId}).then(({initialViewState, mapStyle, layers}) => {
 });
 ```
 
-## Usage
+## Parameters
```

Comment:
I think usage is more clear. What about keeping `Usage` and adding a new `##Parameters` section?